<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelProdDlg</class>
 <widget class="QDialog" name="SelProdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Eating time</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Product (dish)</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="tbMM00">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Quick setting of minutes</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="tbMM15">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Quick setting of minutes</string>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QToolButton" name="tbMM30">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Quick setting of minutes</string>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QToolButton" name="tbMM45">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Quick setting of minutes</string>
     </property>
     <property name="text">
      <string>45</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Weight, g</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="sbWeight">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QLabel" name="lbActInfo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>lbActInfo</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLabel" name="lbProd">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;color:#4800FF; background: #FFF8D0;&quot;&gt; - &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelProdDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelProdDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
